src/main/java/org/akhq/utils/AvroToJsonSerializer.java

```diff
@@ -1,32 +1,49 @@
 package org.akhq.utils;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.TimeZone;
+import javax.inject.Singleton;
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.micronaut.context.annotation.Value;
+import io.micronaut.core.annotation.Nullable;
 import org.apache.avro.generic.GenericRecord;
 
-import java.io.IOException;
-import java.util.Map;
-import java.util.TimeZone;
-
+@Singleton
 public class AvroToJsonSerializer {
-    private static final ObjectMapper MAPPER = new ObjectMapper()
-        .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
-        .setSerializationInclusion(JsonInclude.Include.NON_NULL)
-        .setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
-        .registerModule(new JavaTimeModule())
-        .registerModule(new Jdk8Module())
-        .setTimeZone(TimeZone.getDefault());
+    private final ObjectMapper mapper;
 
-    public static String toJson(GenericRecord record) throws IOException {
-        Map<String, Object> map = AvroDeserializer.recordDeserializer(record);
+    public AvroToJsonSerializer(@Value("${akhq.avro-serializer.json.serialization.inclusions}") @Nullable List<Include> jsonInclusions) {
```

Comment:
You are changing the default configuration, please add old default one on the application.yml please

Reply:
Added the old default to src/main/resources/application.yml

```yaml
  avro-serializer:
    json.serialization.inclusions:
      - NON_NULL
      - NON_EMPTY
 ```